### ISSUE TITLE
XamlDiagnostics is typo'd as "XamlDiganostics"

### ISSUE THREAD

**Issue by reflectronic** [microsoft/win32metadata#193]
It looks like the folder name `generation/Partitions/Xaml_Diganostics` and the lines https://github.com/microsoft/win32metadata/blob/dc071257239b848481dc3aae5dc2c5bf5c76831a/generation/Partitions/Xaml_Diganostics/settings.rsp#L12-L13 both are spelled incorrectly. Seems to be spelled correctly in the namespace remapping csv though, so it puts the `Apis` class into a separate (typo'd) namespace from the actual APIs.

**Comment by reflectronic** [comment]
Also, it seems like the actual C API `InitializeXamlDiagnosticsEx` that makes this thing do anything is missing. The header additionally declares an `InitializeXamlDiagnostic`, but it seems to 1) be misspelled and 2) not exist in any DLLs anymore?

**Comment by sotteson1** [comment]
This is fixed in 10.0.19041.5-preview.2.

**Comment by sotteson1** [comment]
I realized it's still missing InitializeXamlDiagnosticsEx and InitializeXamlDiagnostics. For some reason the scripts aren't find the import libs for them, so they get excluded.

**Comment by sotteson1** [comment]
Fixed in 10.0.19041.5-preview.4.

**Comment by reflectronic** [comment]
The thing is, `InitializeXamlDiagnostic` doesn't exist in `Windows.UI.Xaml.dll`:
```powershell
> $lib = [System.Runtime.InteropServices.NativeLibrary]::Load("Windows.UI.Xaml.dll")
> [System.Runtime.InteropServices.NativeLibrary]::GetExport($lib, "InitializeXamlDiagnostic")
# MethodInvocationException: Exception calling "GetExport" with "2" argument(s): "Unable to find an entry point named 'InitializeXamlDiagnostic' in DLL."
> [System.Runtime.InteropServices.NativeLibrary]::GetExport($lib, "InitializeXamlDiagnosticsEx")
# 140730506599536
